### DISCUSSION TITLE
bulk api - webhooks [May 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2147] [Discord Threads]
> 🙋‍♂️ **kenmt**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### its not supported at the moment  
#  
> 🙋‍♂️ **kenmt**  
#### no problem, just wanted to make sure  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/975799468777697320)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
